### PR TITLE
Sorting dacs and hiding 6 month old projects

### PR DESCRIPTION
**Pull request by arbreton**
https://github.com/Giveth/giveth-dapp/issues/1464

### CODE REVIEW

File: src/services/DACService.js [hunk 1]
```diff
@@ -67,14 +67,20 @@ class DACService {
...
         query: {
+          updatedAt: {
+            $lt: new Date(),
+            $gte: sixMonths,
+          },
           status: DAC.ACTIVE,
           $limit,
           $skip,
-          $sort: { campaignsCount: -1 },
+          $sort: { updatedAt: -1 },
```
Comment (aminlatifi) [inline src/services/DACService.js:83]: It shouldn't be changed

File: src/services/CampaignService.js [hunk 2]
```diff
@@ -38,10 +38,16 @@ class CampaignService {
...
    */
   static getCampaigns($limit = 100, $skip = 0, onSuccess = () => {}, onError = () => {}) {
+    const sixMonths = new Date();
+    sixMonths.setMonth(sixMonths.getMonth() - 6);
     return feathersClient
       .service('campaigns')
       .find({
         query: {
+          updatedAt: {
+            $lt: new Date(),
```
Comment (aminlatifi) [inline src/services/CampaignService.js:48]: What's the purpose of this line? it should be deleted.

File: src/services/DACService.js [hunk 3]
```diff
@@ -67,14 +67,20 @@ class DACService {
...
    */
   static getDACs($limit = 100, $skip = 0, onSuccess = () => {}, onError = () => {}) {
+    const sixMonths = new Date();
+    sixMonths.setMonth(sixMonths.getMonth() - 6);
     return feathersClient
       .service('dacs')
       .find({
         query: {
+          updatedAt: {
+            $lt: new Date(),
```
Comment (aminlatifi) [inline src/services/DACService.js:77]: This should be removed too!

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Comment by arbreton** [comment]
updated PR @aminlatifi

**Review by aminlatifi — APPROVED** [review]

**Comment by aminlatifi** [comment]
It's not working, for example this milestone is update in 2018
https://beta.giveth.io/campaigns/5b37e9daa239ac21b383d4e0

But still is shown after this change

**Review by aminlatifi — CHANGES_REQUESTED** [review]
Doesn't work as expected

**Comment by papermache** [comment]
Still testing.

**Comment by GriffGreen** [comment]
still needs changes

**Comment by aminlatifi** [comment]
> It's not working, for example this milestone is update in 2018
> https://beta.giveth.io/campaigns/5b37e9daa239ac21b383d4e0
> 
> But still is shown after this change

Sorry, it was my mistake, it was working. We have implemented this and put the limit in config in another PR.

**Pull request closed without merging**